client/app/scripts/components/node-details/node-details-table.js

```diff
@@ -74,13 +80,22 @@ function maybeToLower(value) {
 }
 
 
+function toArrayOfThreeDigitStrings(value) {
```

Comment:
Hm.. I really couldn't think of a more fitting name here, do you have any suggestion? :D Test and location are good points, I'm gonna address them now!